quickwit/Cargo.toml

```diff
@@ -118,6 +118,7 @@ num_cpus = "1"
 numfmt = "1.1.1"
 once_cell = "1"
 oneshot = "0.1.5"
+opendal = { version ="0.44", default-features = false }
```

Comment:
not sure I understood why `opendal` is declared here. 
Will it be enough to define it only in the `storage` module?

Reply:
> not sure I understood why `opendal` is declared here.

I found most deps listed in `quickwit` are using `workspace = true` like:

https://github.com/quickwit-oss/quickwit/blob/c4e449ff941c6263386a00cf94c347e853e3f030/quickwit/quickwit-storage/Cargo.toml#L38-L46

I have no preference. I'm happy to change it if you'd like.

Reply:
fair point, let's leave it then 👍